forecasttools/__init__.py

```diff
@@ -28,41 +28,52 @@
 )
 
 # location table (from Census data)
-with importlib.resources.path(
-    __package__, "location_table.parquet"
-) as data_path:
-    location_table = pl.read_parquet(data_path)
+location_table_path = importlib.resources.files(__package__).joinpath(
+    "location_table.parquet"
+)
+location_table = pl.read_parquet(str(location_table_path))
 
 # load example flusight submission
-with importlib.resources.path(
-    __package__, "example_flusight_submission.parquet"
-) as data_path:
-    dtypes_d = {"location": pl.Utf8}
-    example_flusight_submission = pl.read_parquet(data_path)
+example_flusight_submission_path = importlib.resources.files(
+    __package__
+).joinpath("example_flusight_submission.parquet")
+dtypes_d = {"location": pl.Utf8}
+example_flusight_submission = pl.read_parquet(
+    str(example_flusight_submission_path)
```

Comment:
Is the casting to `str` really necessary?

Reply:
Will verify. If not necessary, I will remove. Otherwise, keep.